src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
Just trying to understand this line. As $instance is a declared variable won't it always be set? It may not have a value but that's not the same thing.